lib/src/breakpoint.dart

```diff
@@ -42,6 +42,7 @@ VMBreakpoint newVMBreakpoint(Scope scope, Map json) {
 /// full metadata is always available.
 class VMBreakpoint extends VMObject {
   final Scope _scope;
+  final VMBreakpointLocation _location;
```

Comment:
Nit, but if I have a getter that directly forwards a field, I like to put the field immediately below the getter, e.g.:

```dart
VMBreakpointLocation get location => _location;
final VMBreakpointLocation _location;
```

This keeps related definitions close together and means that you don't need to duplicate documentation.